Parity/main/QwParity.cc

```diff
@@ -275,6 +275,12 @@ Int_t main(Int_t argc, Char_t* argv[])
         //  Send ROC configuration event data to the subsystem objects.
         eventbuffer.FillSubsystemConfigurationData(detectors);
       }
+			// Possible ControlProcess Event Handling
+			// if we decide not to have the Decoder virtually inherit from MQwCodaControlEvent
+			/* if(eventbuffer.IsProccessEvent(){
+ 			*  eventbuffer.ProcessControlEvent();
+ 			*  }
+ 			*/
```

Comment:
// Possible ControlProcess Event Handling
			// if we decide not to have the Decoder virtually inherit from MQwCodaControlEvent
			/* if(eventbuffer.IsProccessEvent(){
 			*  eventbuffer.ProcessControlEvent();
 			*  }
 			*/

It is odd that our decoder class processes control events. Should we create "fControlEventFlag" that we set inside the decoder class, then outside (ie in QwParity) we check for said flag and then process there? 

However, CODA 2 Decoder passes all events with the fIDBankNum = 0xCC but that are not PHYS events through ProcessControlEvent (if they aren't control events, they just drop through). We would need to rethink this logic and the potential issues.